src/Plugin/Field/FieldFormatter/MediaImageFormatter.php

```diff
@@ -50,6 +50,10 @@ public static function preRender($element) {
       }
 
     }
+    // Disable cache for this field formatter.
+    // https://www.drupal.org/project/drupal/issues/2099131.
+    // https://www.drupal.org/node/2151609.
+    $element['#cache'] = FALSE;
```

Comment:
Does something like this work?
```
$element['#cache']['keys'][] = $element['#stanford_media_image_style'];
```
Probably needs to move it up into the `if` statement.

Reply:
Working through: https://www.drupal.org/project/drupal/issues/2469277